### PR TITLE
Fix GitHub Actions issues

### PR DESCRIPTION
**Pull request by devpow112**
Fix issues with `markdownlint`. Removes these warnings. Resolves https://github.com/WNProject/WNFramework/issues/534. Will close https://github.com/WNProject/WNFramework/pull/558.

image

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -123,14 +123,16 @@ jobs:
...
           changes_c_cpp=${changes_c_cpp//$'\n'/'\0'}
-          changes_md=${changes_md//$'\n'/'\0'}
-          echo "::set-output name=run-build::$build"
-          echo "::set-output name=run-check-licenses::$check_licenses"
-          echo "::set-output name=run-format-check-c-cpp::$format_check_c_cpp"
-          echo "::set-output name=run-format-check-md::$format_check_md"
-          echo "::set-output name=changes-externals::$changes_externals"
-          echo "::set-output name=changes-c-cpp::$changes_c_cpp"
-          echo "::set-output name=changes-md::$changes_md"
+          delimiter=$(tr -dc A-Za-z0-9 </dev/urandom | head -c 16)
```
Comment (devpow112) [inline .github/workflows/ci.yml:126]: Using a random delimiter due to this recommendation
https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#multiline-strings

**Comment by devpow112** [comment]
After this merges I'll update all the other repos so I can archive the `docker-markdownlint` repo

**Comment by devpow112** [comment]
This should resolve the issues on the `master` branch

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -123,14 +123,16 @@ jobs:
...
           changes_c_cpp=${changes_c_cpp//$'\n'/'\0'}
-          changes_md=${changes_md//$'\n'/'\0'}
-          echo "::set-output name=run-build::$build"
-          echo "::set-output name=run-check-licenses::$check_licenses"
-          echo "::set-output name=run-format-check-c-cpp::$format_check_c_cpp"
-          echo "::set-output name=run-format-check-md::$format_check_md"
-          echo "::set-output name=changes-externals::$changes_externals"
-          echo "::set-output name=changes-c-cpp::$changes_c_cpp"
-          echo "::set-output name=changes-md::$changes_md"
+          delimiter=$(head -c 16 /dev/urandom | base64)
```
[the remark this answers is not in the document]
Comment (AWoloszyn) [reply .github/workflows/ci.yml:126]: Neat!

**Review by AWoloszyn — APPROVED** [review]

**Pull request merged**